package.json

```diff
@@ -107,8 +107,7 @@
   },
   "lint-staged": {
     "(apps|packages)/**/*.{js,ts,jsx,tsx}": [
-      "prettier --write",
-      "eslint --fix"
+      "prettier --write"
```

Comment:
I couldn't commit because eslint --fix was failing because of an error that had nothing to do with my changes.